README.md

```diff
@@ -16,40 +16,94 @@ Some of these are tricky!  Don't trust your first instinct.
 
 
 **a)** `"1" == 1`
+<pre>
+True. ==  tries to match types 
+</pre>
 
-**b)** `"1" === 1`
+**b)** `"1" === 1` 
+<pre> 
+False, not an exact match.
+</pre>
 
 **c)** `x == 'x'`
+<pre> 
+True, types converted. 
+</pre>
 
 **d)** `x == (x+'')`
+<pre> 
+True, for numbers, I feel like it should be true in general, buts its not! 
```

Comment:
This is actually true in almost all cases--if `x` is a string, adding an empty string will not change it. The few exceptions are booleans, which will be autoconverted to strings when added to a string, and `NaN`, which is not equal to anything, including itself.